mozilla-services/consvc-shepherd
# [SHEP-94] Add name to Campaign Overview in React app

## References

[SHEP-94](https://mozilla-hub.atlassian.net/browse/SHEP-94)
This is the React follow-up PR for https://github.com/mozilla-services/consvc-shepherd/pull/294

## Problem Statement

Campaign Summary does not show campaign name. It shows Advertiser Name; but advertisers can have multiple campaigns and this causes confusion.

## Proposed Changes

Add new field Campaign Name to Campaign Summaries (accomplished in https://github.com/mozilla-services/consvc-shepherd/pull/294).
In this PR, add new name field to the React app.

## Verification Steps

Locally, go to the [Ad Ops Dashboard's Campaign Overview page](http://0.0.0.0:5173/campaign-overview).

Verify that the table has the `name` column.

## Visuals

If applicable, provide screenshots or videos here with the expected new behavior. If not, please remove this section.
<img width="1411" alt="Screenshot 2024-10-29 at 2 14 56 PM" src="https://github.com/user-attachments/assets/0bb106f2-0115-4942-9ccf-0e7ebab2ee00">


[SHEP-94]: https://mozilla-hub.atlassian.net/browse/SHEP-94?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
